compiler/util/src/org/jetbrains/kotlin/utils/threadLocal.kt

```diff
@@ -23,4 +23,6 @@ private class ThreadLocalDelegate<T>(private val initializer: () -> T) : ReadWri
     override operator fun setValue(thisRef: Any?, property: KProperty<*>, value: T) {
         map[Thread.currentThread()] = value
     }
+
+    override fun toString(): String = "ThreadLocalDelegate(map=$map)"
```

Comment:
When a thread is used as a map key, the result of toString is not nice to read 
```suggestion
    override fun toString(): String =
        "ThreadLocalDelegate(${map.entries.joinToString { "#${it.key.id}=>${it.value}" }})"    
```